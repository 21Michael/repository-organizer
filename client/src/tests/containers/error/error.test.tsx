import React from "react";
import { render, cleanup } from "@testing-library/react";
import Error from "../../../containers/error/error";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";
import store from "../../../storeReduxToolkit/configStore";
import { Provider } from "react-redux";
import "mutationobserver-shim";

export default describe("Error: ", () => {
  afterEach(() => {
    cleanup();
  });

  test("error", () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Error />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
