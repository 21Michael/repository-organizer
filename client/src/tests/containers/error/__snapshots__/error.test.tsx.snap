// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error:  error 1`] = `
<div>
  <div
    class="error"
  >
    Error 404
    <br />
    Page wasn't found
  </div>
</div>
`;
